srcStatic/GameModule.h

```diff
@@ -35,6 +35,11 @@ class GameModule
 	// Console module calls: mod_run
 	virtual void Run() { };
 
+	// Returns the directory associated with this module
+	// Can be used to access additional resources packaged with the module
+	// By default return the empty string (implying no additional resources)
+	virtual std::string Directory() { return std::string(); }
```

Comment:
I'm not sure that I prefer this over the dynamic_cast, but not too big a deal either way...